File: docs/legacy(old PACE)/C#/PaceAPISample/sample/ReadJobShipment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using efipaceservices;

namespace Pace_Web_Service_SDK.sample
{
    class ReadJobShipment
    {
        public static void Run()
        {
            int id = SDK.readInputFromTheUser("Please provide job shipment id to be read : ");
            ReadObjectPortType readObjectHttpBinding = PaceClient.getReadObjectHttpBinding();
            JobShipment jobShipment = new JobShipment();

            jobShipment.id = id;
            try
            {
                jobShipment = readObjectHttpBinding.readJobShipment(jobShipment);
                Console.WriteLine("Job Shipment : " + jobShipment.id + " was read successfully");
            }
            catch(Exception e)
            {
                Console.WriteLine("JobShipment: " + id + " does not exist. " + e.Message);
            }
        }
    }
}

File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/sample/ReadJobShipment.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class ReadJobShipment
    Public Shared Sub Run()

        Dim id As Integer = 5150
        Dim readObjectHttpBinding As ReadObjectHttpBinding = PaceClient.getReadObjectHttpBinding()
        Dim jobShipment As JobShipment = New JobShipment()

        jobShipment.id = id
        Try
            jobShipment = readObjectHttpBinding.readJobShipment(jobShipment)
            Console.WriteLine("Job Shipment : " + jobShipment.id + " was read successfully")
        Catch ex As Exception
            Console.WriteLine("JobShipment: " + id + " does not exist. " + ex.Message)
        End Try

    End Sub


End Class
